### ISSUE TITLE
Getting: "This message contains an event that this version of Lightning cannot process."

### ISSUE THREAD

**Issue by Paddler21** [Ericsson/exchangecalendar#432]
If I setup a meeting inviting external email addresses and receive their ACCEPT the accept message is always shown with: ""This message contains an event that this version of Lightning cannot process." Is this a known issue or can this be fixed? Thanks!
TB 38.6.0 Lightning 4.0.6 Exchange EWS Provider 3.6.0-beta1
Exchange 14.3.248.2 client access server 14.3.123.0

**Comment by bavincen** [comment]
This can be fixed .. can you help me finding right api.
You can discuss this in lightning..

Regards

**Comment by Paddler21** [comment]
Hm, not sure what you mean exactly - should I report it in the Lightning forum?

**Comment by bavincen** [comment]
I need your help. If you are developer you can help me to find  proper
lightning function which will  display correct message. You can Google or
talk to the lightning communities

**Comment by Ekmansoft** [comment]
I see this when running Thunderbird/Lightning and Icedove (45.5.1) and Iceowl (4.7.5.1) on Debian still when receiving invitations made from Outlook... 
Is there a fix for it or a thread that I could join to know what is happening?
[meeting-invite-outlook.txt](https://github.com/Ericsson/exchangecalendar/files/660987/meeting-invite-outlook.txt)

**Comment by freechelmi** [comment]
I don't think this is specific to external address, answer from internal outlook generate the same warning

**Comment by joefresco** [comment]
I see this when someone sends an invite from Apple Calendar, originating from a noreply@email.apple.com address.

Tbird 52.9.1 and Lightning 5.4.9.1
